### PR TITLE
Add generic logger trait and helper

### PR DESCRIPTION
**Pull request by dumega**

### CODE REVIEW

File: app/code/community/Bolt/Boltpay/Helper/LoggerTrait.php [hunk 1]
```diff
@@ -0,0 +1,147 @@
+<?php
+
+/**
+ * Describes log levels.
+ */
+class Bolt_Boltpay_Helper_LogLevel
+{
+    const EMERGENCY = 'emergency';
```
Comment (daisy1754) [inline app/code/community/Bolt/Boltpay/Helper/LoggerTrait.php]: I suggest we only do error, warning and inof for now

Comment (mccottry) [reply app/code/community/Bolt/Boltpay/Helper/LoggerTrait.php]: @dumega I agree with Kazuki in that I don't 'emergency', 'alert', and 'notice' as commonly used level in things like log4J and its derivatives.  

@daisy1754 If we do cover all of the common logger levels, perhaps in the concrete classes, the implementation of the other levels can be mapped to the three you listed above, or in cases like 'trace' ignored by some loggers like bugsnag.

**Comment by daisy1754** [comment]
@mccottry actually I was thinking datadog as a concrete implementation

File: app/code/community/Bolt/Boltpay/Helper/LoggerTrait.php (hunk 1 above)
Comment (dumega) [reply app/code/community/Bolt/Boltpay/Helper/LoggerTrait.php]: @daisy1754 I update it, and we can easily extend the log level in the future

**Comment by dumega** [comment]
@mccottry yes, I just thought we may use different logger for different workaround, and traits is good for that

**Pull request merged**